chucks-pick3-web/angular-app/src/app/components/home.component/home.component.html

```diff
@@ -9,7 +9,7 @@
   <div class="col-md-4 no-pad-right">
     <a [routerLink]="['/requirements']" [queryParams]="{showSubNavBar: true}">
       <div class="grediant">
-        <img src="assets/images/screens-02.png">
+        <img src="src/app/assets/images/screens-02.png">
```

Comment:
Not Sure about this change.  Does this still work?  I think it is okay to start paths to images and css files with just `assets/images` and `assets/css`;  Please revert back.

Reply:
reverted to assets/images